setup.py

```diff
@@ -17,50 +17,15 @@
 #   limitations under the License.
 #
 # ------------------------------------------------------------------------------
-import importlib
 import os
 import re
-from typing import Dict, List
+from typing import Dict
 
 from setuptools import find_packages, setup
 
 PACKAGE_NAME = "aea"
 
 
-def get_aea_extras() -> Dict[str, List[str]]:
-    """Parse extra dependencies from aea channels and protocols."""
-    result = {}
-
-    # parse connections dependencies
-    connection_module = importlib.import_module("aea.connections")
-    connection_dependencies = {
-        k.split("_")[0] + "-connection": v
-        for k, v in vars(connection_module).items()
-        if re.match(".+_dependencies", k)
-    }
-    result.update(connection_dependencies)
-
-    # parse protocols dependencies
-    protocols_module = importlib.import_module("aea.protocols")
-    protocols_dependencies = {
-        k.split("_")[0] + "-protocol": v
-        for k, v in vars(protocols_module).items()
-        if re.match(".+_dependencies", k)
-    }
-    result.update(protocols_dependencies)
-
-    # parse skills dependencies
-    skills_module = importlib.import_module("aea.skills")
-    skills_dependencies = {
-        k.split("_")[0] + "-skill": v
-        for k, v in vars(skills_module).items()
-        if re.match(".+_dependencies", k)
-    }
-    result.update(skills_dependencies)
-
-    return result
-
-
```

Comment:
quite obsolete now

Reply:
are we sure we don't want to keep it around?

Reply:
don't know; Anyway, if we will really need many default packages, I think it is better to retrieve the dependencies from their `yaml` file instead of this approach.